coalition/index.html

```diff
@@ -88,37 +88,37 @@
 		</nav>
 
 		<header data-aos="fade-up" style="background-image: url('../assets/images/coalition.png'); background-size: cover; background-repeat: no-repeat; min-height: 120vh;background-position:top;">
-		<div class="container-fluid;">
-			<div class="row" style="padding-top:30vh">
-				<div class="col-1 col-md-6"></div>
-				<div class="col-10 col-md-4">
-					<h1>The Impact Coalition</h1>
-					<p class="lead">
-						We match you with a company that truly makes a difference.
-						<br/>
-						<br/>  
-						Applications for The 2019 Impact Coalition are now closed. <a target="_blank" href="https://impactlabs.us19.list-manage.com/subscribe/post?u=e271c51cea784ba1c111d990e&amp;id=90db606254">Sign up for our newsletter</a> to find out when they open for 2020!
-					</p>
+			<div class="container-fluid;">
+				<div class="row" style="padding-top:30vh">
+					<div class="col-1 col-md-6"></div>
+					<div class="col-10 col-md-4">
+						<h1>The Impact Coalition</h1>
+						<p class="lead">
+							We match you with a company that truly makes a difference.
+							<br/>
+							<br/>  
+							Applications for The 2019 Impact Coalition are now closed. <a target="_blank" href="https://impactlabs.us19.list-manage.com/subscribe/post?u=e271c51cea784ba1c111d990e&amp;id=90db606254">Sign up for our newsletter</a> to find out when they open for 2020!
+						</p>
+					</div>
+					<div class="col-1 col-md-2"></div>
```

Comment:
not immediately clear what's this for

Reply:
It adds the proper column spacing, or else the text would stretch the whole width of the screen.